### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by collierrgbsitisfise**
Was fixed minor type: 'Лозволяющая' -> 'Позволяющая'.
Actually i would not recommend to translate it word into word. MB here is better to use word - 'Гибкая' (flexible) instead of 'Позволяющая'.

### CODE REVIEW

**Comment by goldbergyoni** [comment]
@collierrgbsitisfise Welcome aboard!

@contributorpw Have a look?

**Comment by goldbergyoni** [comment]
Waiting for our Russian expert @contributorpw

File: sections/errorhandling/useonlythebuiltinerror.russian.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### Объяснение в один абзац
 
-Лозволяющая природа JavaScript наряду с его разнообразием параметров потока кода (например, EventEmitter, Callbacks, Promises и т.д.) приводит к значительному расхождению в том, как разработчики выдают ошибки - некоторые используют строки, другие определяют свои собственные пользовательские типы. Использование встроенного объекта Error в Node.js помогает сохранить единообразие в вашем коде, а с помощью сторонних библиотек он также сохраняет важную информацию, такую ​​как StackTrace. При возникновении исключения обычно рекомендуется заполнить его дополнительными контекстными свойствами, такими как имя ошибки и связанный код ошибки HTTP. Чтобы добиться этого единообразия и практики, рассмотрите возможность расширения объекта Error дополнительными свойствами, см. пример кода ниже.
```
Comment (contributorpw) [inline sections/errorhandling/useonlythebuiltinerror.russian.md]: Вероятно, "гибкая", действительно, будет лучше. Не могли бы вы это добавить?

**Review by contributorpw — DISMISSED** [review]

**Comment by goldbergyoni** [comment]
@contributorpw Do you have merge rights? do you want to have? I think you should

**Comment by goldbergyoni** [comment]
@collierrgbsitisfise

The ball is at your court, let us know if we can help somehow

**Comment by goldbergyoni** [comment]
@all-contributors please add @mindock for content

**Comment by goldbergyoni** [comment]
@collierrgbsitisfise Are you up to fixing the merge conflict? This would be a great help 🙏

**Comment by collierrgbsitisfise** [comment]
@goldbergyoni,  hey sorry , my bad - forgot about this PR. Done ! I guess it's ready to be merged

**Comment by kevynb** [comment]
Were the requested changes taken into account? If so I can merge this PR

**Comment by collierrgbsitisfise** [comment]
@contributorpw are we ok to merge this one ?

**Comment by kevynb** [comment]
Since he doesn't seems available let merge that anyway, this typo has been here for far too long 😉 

@all-contributors please add @collierrgbsitisfise for translation

**Review by kevynb — APPROVED** [review]

**Pull request merged**